vault/resource_kubernetes_secret_backend_test.go

```diff
@@ -0,0 +1,136 @@
+					resource.TestCheckResourceAttr(resourceName, fieldKubernetesHost, ""),
+					resource.TestCheckResourceAttr(resourceName, fieldKubernetesCACert, ""),
+					resource.TestCheckResourceAttr(resourceName, fieldDisableLocalCAJWT, "false"),
+				),
+			},
+			{
+				Config: testKubernetesSecretBackend_mountTuneConfig(path),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, consts.FieldPath, path),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldDescription, "kubernetes secrets engine"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldDefaultLeaseTTL, "3600"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldMaxLeaseTTL, "7200"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldLocal, "true"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldAuditNonHMACRequestKeys+".0", "test_req"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldAuditNonHMACResponseKeys+".0", "test_res"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldSealWrap, "true"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldExternalEntropyAccess, "true"),
+					resource.TestCheckResourceAttr(resourceName, fieldKubernetesHost, ""),
+					resource.TestCheckResourceAttr(resourceName, fieldKubernetesCACert, ""),
+					resource.TestCheckResourceAttr(resourceName, fieldDisableLocalCAJWT, "false"),
+				),
+			},
+			{
+				Config: testKubernetesSecretBackend_updateConfig(path),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, consts.FieldPath, path),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldDescription, "kubernetes secrets description updated"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldDefaultLeaseTTL, "0"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldMaxLeaseTTL, "0"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldLocal, "false"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldAuditNonHMACRequestKeys+".#", "0"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldAuditNonHMACResponseKeys+".#", "0"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldSealWrap, "false"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldExternalEntropyAccess, "false"),
+					resource.TestCheckResourceAttr(resourceName, fieldKubernetesHost, "https://127.0.0.1:63247"),
+					resource.TestCheckResourceAttr(resourceName, fieldKubernetesCACert, "test_ca_cert"),
+					resource.TestCheckResourceAttr(resourceName, fieldDisableLocalCAJWT, "true"),
+				),
+			},
+			{
+				ResourceName:            resourceName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{fieldServiceAccountJWT},
+			},
+		},
+	})
+}
+
+func testAccKubernetesSecretBackendCheckDestroy(path string) resource.TestCheckFunc {
```

Comment:
Suggest replacing with https://github.com/hashicorp/terraform-provider-vault/blob/c520061319a277600ea367adacdcf7e17d2820ee/vault/testhelpers_test.go#L14

Reply:
Cool! Done in ea2dc71.